### DISCUSSION TITLE
How to get an Output from 'DispatchWorkflow' activity in Elsa Workflow?

### DISCUSSION THREAD

**Discussion by Manali-23** [elsa-workflows/elsa-core#4666] [Q&A]
I'm trying to get the output from a dispatched workflow at the main workflow. Can u please explain what are the steps to do that
@sfmskywalker

**Comment by sfmskywalker** [comment]
Thank you for your question, this prompted me to write [a new guide about this very topic](https://v3.elsaworkflows.io/docs/guides/dispatching-child-workflows). Please have a look and let me know if this works for you or if anything is unclear still.
